generator/operation.go

```diff
@@ -1076,3 +1075,40 @@ func (b *codeGenOpBuilder) setBodyParamValidation(p *GenParameter) {
 	}
 
 }
+
+// makeSecuritySchemes produces a sorted list of security schemes for this operation
+func (b *codeGenOpBuilder) makeSecuritySchemes(receiver string) GenSecuritySchemes {
+	return gatherSecuritySchemes(b.SecurityDefinitions, b.Name, b.Principal, receiver)
+}
+
+// makeSecuritySchemes produces a sorted list of security requirements for this operation.
+// As for current, these requirements are not used by codegen (sec. requirement is determined at runtime).
+// We keep the order of the slice from the original spec, but sort the inner slice which comes from a map,
+// as well as the map of scopes.
+func (b *codeGenOpBuilder) makeSecurityRequirements(receiver string) []GenSecurityRequirements {
+	if b.Security == nil {
+		if b.Doc.Spec().Security == nil && b.Operation.Security == nil {
+			// work around bug in analysis.SecurityRequirementsFor()
+			// TODO: remove second check this when fixed in analysis
+			// nil (default requirement) is different than [] (no requirement)
```

Comment:
The changes in analysis were merged, you can remove this

Reply:
Yes: that was the plan...

I would like that we update vendor beforehand. I can take care of that. Would you like to merge the pending PR in spec before that?